buildSrc/src/main/resources/plugin-descriptor.properties

```diff
@@ -49,6 +49,9 @@ java.version=${javaVersion}
 opensearch.version=${opensearchVersion}
 ### optional elements for plugins:
 #
+# 'folderName': the custom name of the folder in which the plugin is installed.
+folderName=${folderName}
```

Comment:
How about renaming this to `custom.directory.name`  to be intuitive and consistent with existing pattern ?

Reply:
Is the `.directory.` used anywhere else? That's unusual for a `path`.

Reply:
How about `custom.foldername`?

Reply:
👍

Reply:
Renamed it.